src/main/kotlin/lotto/LottoChecker.kt

```diff
@@ -0,0 +1,37 @@
+package lotto
+
+class LottoChecker {
+
+    fun lottoCheck(winningNumber: String, lottoBundle: List<List<Int>>): List<Int> {
```

Comment:
함수명만 봐서는 로또의 무엇을 체크하는지 와닿지가 않네요. 🤔 
함수가 하는 일에 대해서 좀 더 의도가 드러나도록 함수명을 변경해보면 어떨까요?

Reply:
```suggestion
    fun lottoCheck(winningNumber: String, lottoBundle: List<List<Int>>): List<LottoRank> {
```
LottoRank 클래스는 로또번호 일치 개수에 대해서 알고 있습니다.
그렇다면 LottoRank 를 활용해보면 어떨까요?

Reply:
LottoRank 로 수정하였습니다~! 해당 과정에서 변경되는 코드 및 테스트 코드 수정하였습니다!